### DISCUSSION TITLE
Using UERANSIM with other stacks

### DISCUSSION THREAD

**Discussion by auandrew** [aligungr/UERANSIM#306] [Q&A]
Hello @aligungr & UERANSIM contributors,
Could I know whether UERANSIM can work with other open source 5G core stacks other than open5gs?  And whether anybody has tried & successfully done that before?  And, if so, what that stack is please?
Thanks in advance,
Andrew

**Comment by aligungr** [comment]
Hello Andrew

I personally tested and integrated with 6 different core networks. 2 of them are open source and 4 of them are commercial.

Open sources ones are:
- Open5gs
- Free5GC

For free5gc, you can even find an example configurations in our repository.

Regards

**Comment by auandrew** [comment]
Hello Ali, that's great to know.  
Hello contributors, anybody who has tried UERANSIM with the open source oai-cn5g yet, may I know?
Thanks!
